lib/pages/home_page.dart

```diff
@@ -10,6 +13,28 @@ class HomePage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    if (context.read<PlaylistStore>().playlists.isEmpty) {
```

Comment:
This is not reactive. Once `.playlists` are not empty this widget will not update